eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl

```diff
@@ -37,9 +37,12 @@ SOFTWARE.
               <xsl:attribute name="code">
                 <xsl:value-of select="text()"/>
               </xsl:attribute>
-              <xsl:attribute name="loc">
+              <xsl:attribute name="insert_loc">
```

Comment:
@levBagryansky Do we need `insert_loc` somewhere? I can't find the usage.

Reply:
@volodya-lombrozo it is not used anymore. But it can be usefull as additional information for user.